plugins/palo_alto_cortex_xdr/help.md

```diff
@@ -56,6 +56,46 @@ Example input:
 
 ### Actions
 
+#### Get XQL Query Results
+
+This action is used to start an XQL query and retrieve the query results.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|from|integer|None|False|Integer in timestamp epoch milliseconds for end of the time range, Cortex XDR calls by default the last 24 hours if both 'To' and 'From' values are not present|None|1599080399000|
+|limit|integer|20|False|Integer representing the maximum number of results to return, defaults to 20, max value 1000|None|100|
+|query|string|None|True|String of the XQL query|None|dataset=xdr_data | fields event_id, event_type, event_sub_type | limit 3|
+|tenants|[]string|None|True|String that represents additional information regarding the action|None|["tenantID", "tenantID"]|
+|to|integer|None|False|Integer in timestamp epoch milliseconds for start of the time range, Cortex XDR calls by default the last 24 hours if both 'To' and 'From' values are not present|None|1598907600000|
+
+Example input:
+
+```
+{
+  "from": 1599080399000,
+  "limit": 100,
+  "query": "dataset=xdr_data | fields event_id, event_type, event_sub_type | limit 3",
+  "tenants": [
+    "tenantID",
+    "tenantID"
+  ],
+  "to": 1598907600000
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|reply|reply|False|Was the operation successful|
+
+Example output:
+
+```
+```
```

Comment:
Is there an output for this action, even a simple 'True/Success' etc?

Reply:
Have updated now with an example output.